python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1140
   - name: u_blendedness_flag
     "@id": "#Object.u_blendedness_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1141
   - name: u_cModelFlux
     "@id": "#Object.u_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1142
   - name: u_cModelFluxErr
     "@id": "#Object.u_cModelFluxErr"
     datatype: double
     description: Flux uncertainty from the final cmodel fit. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1143
   - name: u_cModelFlux_inner
     "@id": "#Object.u_cModelFlux_inner"
     datatype: double
     description: Flux within the fit region, with no extrapolation. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1144
   - name: u_cModel_flag
     "@id": "#Object.u_cModel_flag"
     datatype: boolean
     description: Flag set if the final cmodel fit (or any previous fit) failed. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1145
   - name: u_cModel_flag_apCorr
     "@id": "#Object.u_cModel_flag_apCorr"
     datatype: boolean
     description: Set if unable to aperture correct modelfit_CModel. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1146
   - name: u_calibFlux
     "@id": "#Object.u_calibFlux"
     datatype: double
     description: Flux within 12.0-pixel aperture. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1147
   - name: u_calibFluxErr
     "@id": "#Object.u_calibFluxErr"
     datatype: double
     description: Flux uncertainty within 12.0-pixel aperture. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1148
   - name: u_calibFlux_flag
     "@id": "#Object.u_calibFlux_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1149
   - name: u_calibFlux_flag_apertureTruncated
     "@id": "#Object.u_calibFlux_flag_apertureTruncated"
     datatype: boolean
     description: Aperture did not fit within measurement image. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1150
   - name: u_calibFlux_flag_sincCoeffsTruncated
     "@id": "#Object.u_calibFlux_flag_sincCoeffsTruncated"
     datatype: boolean
     description: Full sinc coefficient image did not fit within measurement image. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1151
   - name: u_calib_astrometry_used
     "@id": "#Object.u_calib_astrometry_used"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1152
```

Comment:
These `<band>_calib` flags should probably go up with the other general-purpose flags like the `detect_` ones, rather than be mixed in here with the photometry algorithms.